2-ui/5-loading/03-onload-onerror/article.md

```diff
@@ -69,12 +69,12 @@ script.onerror = function() {
 */!*
 ```
 
-Please note that we can't get HTTP error details here. We don't know was it error 404 or 500 or something else. Just that the loading failed.
+Please note that we can't get HTTP error details here. We don't know if it was an error 404 or 500 or something else. Just that the loading failed.
 
 ```warn
 Events `onload`/`onerror` track only the loading itself.
 
-Errors during script processing and execution are out of the scope of these events. To track script errors, one can use `window.onerror` global handler.
+Errors during script processing and execution are out of scope for these events. To track script errors, one can use `window.onerror` global handler.
```

Comment:
I'm not sure about this one. It depends on whether "out of scope" has a specific meaning in this context (like "out of range for this array" or something). Otherwise, if is a matter of something more like "not included in these events" I prefer the original.

Reply:
I mean: these events do not trigger for such kinds of errors. Which variant is better?

Reply:
Probably the original. Or you could say "Errors during script processing and execution do not trigger these events."